<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>1006</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="UartSettingGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>串口设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="UartPortLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>串口号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="BaudLabel">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="StopBitLabel">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="DataBitLabel">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="ParityBitLabel">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="OpenUartPushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="UartPortComboBox"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="BaudComboBox"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="StopBitComboBox"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="DataBitComboBox"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="ParityBitComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="ReceiveSettingGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>接收设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="ReceiveTextRadioButton">
      <property name="text">
       <string>文本</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="ReceiveHexRadioButton">
      <property name="text">
       <string>Hex</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="AutoNewLineCheckBox">
      <property name="text">
       <string>自动换行</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="ReceiveDIsplayCheckBox">
      <property name="text">
       <string>显示接收</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="ClearReceivePushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清除显示</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="SendSettingGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>发送设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="SendTextRadioButton">
        <property name="text">
         <string>文本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SendHexRadioButton">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1,10">
      <item>
       <widget class="QCheckBox" name="AutoRepeatCheckBox">
        <property name="text">
         <string>自动重发(ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>36</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="RepeatNumberLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="ClearSendPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>清除发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SendPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="WaveSettingGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>730</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>波形设置</string>
   </property>
   <widget class="QPushButton" name="WaveDataExportPushButton">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>154</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据导出</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>36</y>
      <width>231</width>
      <height>105</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="WaveChanalOneCheckBox">
         <property name="text">
          <string>通道一</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="WaveChanalTwoCheckBox">
         <property name="text">
          <string>通道二</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="WaveChanalThreeCheckBox">
         <property name="text">
          <string>通道三</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="WaveChanalFourCheckBox">
         <property name="text">
          <string>通道四</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="WaveChanalFiveCheckBox">
         <property name="text">
          <string>通道五</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="WaveChanalSixCheckBox">
         <property name="text">
          <string>通道六</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SendGroupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>880</y>
     <width>971</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>发送内容</string>
   </property>
   <widget class="QPlainTextEdit" name="SendPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>951</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ReceiveGroupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>971</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>接收内容</string>
   </property>
   <widget class="QTextBrowser" name="ReceiveTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>951</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="WaveGroupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>971</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>波形显示</string>
   </property>
   <widget class="QChartView" name="WaveDisplayGraphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>951</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
